assets/models/organelles/BindingAgent.tscn

```diff
@@ -0,0 +1,37 @@
+[gd_scene load_steps=9 format=2]
+
+[ext_resource path="res://shaders/Organelle.shader" type="Shader" id=1]
+[ext_resource path="res://assets/textures/BA_Fibers_Albedo.png" type="Texture" id=2]
+[ext_resource path="res://assets/textures/BA_Membrane_AO.png" type="Texture" id=3]
+[ext_resource path="res://assets/textures/BA_Membrane_specular.png" type="Texture" id=4]
+[ext_resource path="res://assets/models/organelles/BA_Membrane.mesh" type="ArrayMesh" id=5]
+[ext_resource path="res://assets/models/organelles/BA_Fibers.mesh" type="ArrayMesh" id=6]
```

Comment:
I think all of these file names should be spelled out as "BindingAgent" instead of using BA abbreviation because all the other organelle models and textures are.